buildbot-host/buildmaster/openvpn/tserver_null_pre_steps.cfg

```diff
@@ -0,0 +1,13 @@
+# -*- python -*-
+# ex: set filetype=python:
+def openvpnAddTServerNullPreStepsToBuildFactory(factory, combo, shell_env=None):
+    if shell_env is None:
+        shell_env={}
+
+    factory.addStep(steps.ShellCommand(command=["touch", "tests/t_server_null.rc"],
+                                        env=shell_env,
+                                        name="touch t_server_null.rc",
```

Comment:
This is not enough for the standalone workers. They usually run as a non-privileged user, so they need to set `RUN_SUDO`.

Reply:
Perhaps we could just do what we do with t_client.rc:

```
    factory.addStep(
        steps.ShellCommand(
            command=["cp", "-f", "/buildbot/t_server_null.rc", "."],
            name="copy t_server_null.rc",
            description="copying",
            descriptionDone="copying",
        )
    )
```

For Docker workers we'd bake that config in. For standalone workers we'd have RUN_SUDO in it.

Reply:
Well, actually, if we keep the "touch" BuildStep we don't need to bake an empty file into the image for most buildbot workers.

Reply:
I pushed a change that allows customizing t_server_null.rc on a per-worker basis. A default file is provided for each worker. All the container images need to be rebuilt which takes a bit of time.

I've tested the change and it seems to work just as you'd expect it to.